config/main.py

```diff
@@ -1547,6 +1547,15 @@ def reload(db, filename, yes, load_sysinfo, no_service_restart, force, file_form
         if len(cfg_files) != num_cfg_file:
             click.echo("Input {} config file(s) separated by comma for multiple files ".format(num_cfg_file))
             return
+        
+        # Check if the file is properly formatted before proceeding.  
+        for file in cfg_files:
+            try:
+                read_json_file(file)
+            except Exception as e:
+                click.secho("Bad format: json file broken. {}".format(str(e)),
+                            fg='magenta')
+                sys.exit(1)
```

Comment:
Should we also verify formatting of `DEFAULT_CONFIG_DB_FILE` in case user doesn't give the file path/name?

Reply:
Updated the verify step so it validates all config files used